services/accounts/accounts.go

```diff
@@ -211,6 +211,18 @@ func (api *API) AddKeypair(ctx context.Context, password string, keypair *accoun
 	return nil
 }
 
+func (api *API) RemainingAccountCapacity(ctx context.Context) (int, error) {
+	return (*api.messenger).RemainingAccountCapacity()
+}
+
+func (api *API) RemainingKeypairCapacity(ctx context.Context) (int, error) {
```

Comment:
We may want to more diligently document our public API. The name of the endpoints are usually self-explanatory, but they don't give any recommendation about when to use, why, potential concerns, etc. Could be really helpful for a developer without context.

Some level of documentation on the public API could be our minimum standard in the future, something for the status-go council to think about :) cc @igor-sirotin @Samyoul 

If you think there's nothing too interesting to document it's okay 👍🏼